share/spice/movies/cast/triggers.txt

```diff
@@ -0,0 +1,33 @@
+new movies with
```

Comment:
We could also build these in the Perl with an array or two and a `map` -- either approach is fine really